src/main/java/org/example/persistence/sql/SQLDialect.java

```diff
@@ -12,6 +12,9 @@ public class SQLDialect {
     public static final String SQL_FIND_ALL = """
             SELECT * FROM
             """;
+    public static final String UPDATE_STUDENT= """
+            UPDATE students set first_name = ? WHERE id = ?
```

Comment:
Please see getTableAndColumnNamesForInsert and getTableNameForSelect methods in SQLDialect class, and create similar method (getTableAndColumnNamesForUpdate or something like that) for update so you can remove the hardcoded table name and columns in the update statement.